BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -194,7 +194,7 @@ def create_stochastic_schedules(args:,
     sinks = random_shift_and_normalize(sink_activity_sch, random_offset)
     baths = random_shift_and_normalize(bath_activity_sch, random_offset)
     fixtures = [showers, sinks, baths].transpose.map(&:sum)
-    @schedules[SchedulesFile::Columns[:HotWaterFixtures].name] = fixtures.map { |flow| flow / fixtures.max }
+    @schedules[SchedulesFile::Columns[:HotWaterFixtures].name] = normalize(fixtures)
```

Comment:
This is the primary bugfix. `fixtures.max` was being calculated within the iteration resulting in a huge performance hit.